integration/resource_lacework_alert_channel_datadog_test.go

```diff
@@ -22,7 +29,8 @@ func TestDatadogAlertChannelCreate(t *testing.T) {
 
 	// Update Datadog Alert Channel
 	terraformOptions.Vars = map[string]interface{}{
-		"channel_name": "Datadog Alert Channel Updated"}
+		"channel_name": "Datadog Alert Channel Updated",
+		"api_key":      apiKey}
```

Comment:
Could we do two things:
1. update the fields that we have above to verify that "updates" are working fine
2. assert more fields and not only the integration name, things like the channel name, site, service and API key
```suggestion
		"api_key":      "updated-fake-key"}
```